lib/anaconda/form_builder_helpers.rb

```diff
@@ -94,7 +100,7 @@ def anaconda( anaconda_field_name, form_options = {} )
         attribute: "#{options[:as]}"
       });
     }).call(this);
-    });
+    })
```

Comment:
Probably should outdent this so it lines up with the `d` in `document...`

Reply:
I should probably start this PR over at this point...Monday problems...